plugins/zoom/help.md

```diff
@@ -125,125 +126,103 @@ Example input:
   "transfer_recordings": false,
   "transfer_webinars": false
 }
-```
+```  
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|success|boolean|True|Success|
+|Name|Type|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- |
+|success|boolean|True|Success|True|
 
 Example output:
 
 ```
 {
   "success": true
 }
-```
+```  
 
 #### Get User
-
-This action is used to get user in Zoom account.
+  
+Get user in Zoom account  
 
 ##### Input
 
 |Name|Type|Default|Required|Description|Enum|Example|
-|----|----|-------|--------|-----------|----|-------|
+| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
 |user_id|string|None|True|The user identifier or email address|None|user@example.com|
-
+  
 Example input:
 
 ```
 {
   "user_id": "user@example.com"
 }
-```
+```  
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|user|user|True|User details|
+|Name|Type|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- |
+|user|user|True|User details|{}|
 
 Example output:
 
 ```
 {
-  "user": {
-    "account_id": "",
-    "role_name": "",
-    "id": "oZSx8rYaTPW8Ibjzy8nKFA",
-    "im_group_ids": [],
-    "phone_country": "",
-    "type": 1,
-    "verified": 0,
-    "created_at": "2020-06-03T21:30:52Z",
-    "first_name": "",
-    "language": "",
-    "phone_number": "",
-    "group_ids": [],
-    "last_name": "",
-    "status": "pending"
-  }
+  "user": {}
```

Comment:
We lost a couple of examples here. They are all need to be added to `plugin.spec.yaml` file